kubespawner/spawner.py

```diff
@@ -324,6 +328,28 @@ async def async_method(self, *args, **kwargs):
         """,
     )
 
+    user_namespace_labels = Dict(
+        {},
+        config=True,
+        help="""
+        Kubernetes labels that user namespaces will get.
+
+        Note that these are only set when the namespaces are created, not
+        later when this setting is updated.
+        """,
+    )
+
+    user_namespace_annotations = Dict(
+        {},
```

Comment:
I'm not sure, but I think a single `{}` object could be the same object for all spawner instances, which is problematic if its modified later for one spawner instance but not others.

Maybe there is some example of using the `Dict` traitlets configuration object somewhere else you can mimic to avoid this kind of issue.

Reply:
I wasn't aware that `{}` could refer to the same object in all instances, fixed and thank you for the info!